src/firebase/firestore/job.ts

```diff
@@ -108,14 +111,33 @@ export const getAllJobs = async (collectionName: string): Promise<Job[]> => {
   }
 };
 
-// export const updateJob = async (job_id: string): Promise<void> => {};
+export const approveOrDeclineJob = async (
```

Comment:
for the functions approveOrDeclineJob and deleteJob I would say to just have a single deleteJob that takes in a collection name and deletes from a particular collection. then make the rest of the logic of deleteJob (of removing from all users bookmarked collection) to another function